tachyon/init.sh

```diff
@@ -13,6 +13,9 @@ then
   # Not yet supported
   echo "Tachyon git hashes are not yet supported. Please specify a Tachyon release version."
 # Pre-package tachyon version
+elif [[ "$HADOOP_MINOR_VERSION" == "2.6" ]]
```

Comment:
Also need to handle 2.7 ? I actually think we can turn off tachyon if Spark version >= 2.0 / or hadoop version = yarn.